wiki/Tournaments/RESC/2025/en.md

```diff
@@ -0,0 +1,162 @@
+---
+tags:
+  - ResCup
+  - Res Cup
+  - Serenity
+  - Regulus
+  - Regulus Revolution
+---
+
+# Resurrection Cup 2025
+
+![Resurrection Cup 2025](/wiki/shared/news/2025-05-11-resurrection-cup-2025-registrations-now-open/banner.jpg)
```

Comment:
other way round - for shared banners between news and wiki articles, they've always gone in the wiki article's img folder